cmd/activator/main.go

```diff
@@ -278,16 +278,13 @@ func main() {
 		}(name, server)
 	}
 
-	// Enable TLS server when internal-encryption is specified.
+	// Enable TLS server when DataPlaneTrust is not netcfg.TrustDisabled.
 	// At this moment activator with TLS does not disable HTTP.
 	// See also https://github.com/knative/serving/issues/12808.
 	if tlsEnabled {
 		name, server := "https", pkgnet.NewServer(":"+strconv.Itoa(networking.BackendHTTPSPort), ah)
 		go func(name string, s *http.Server) {
-			s.TLSConfig = &tls.Config{
-				MinVersion:     tls.VersionTLS12,
-				GetCertificate: certCache.GetCertificate,
-			}
+			s.TLSConfig = &certCache.ServerTLSConf
```

Comment:
A better option may be to simply add the new CA certificate to the existing pool since we are already using GetCertificates, this may be enough. 

(tlsCOnf cloning is a shallow cloning of the tlsConf). 
I will try and test

Reply:
> I consider an alternative to close the listener and recreate it upon TLSConfig updates.

This would drop traffic and I don't think we should do this

Reply:
We are now adding the new CA to the existing pool instead of creating a new pool. Since cloning is shallow this should solve the problem.